style.css/firstsectionStyle.css

```diff
@@ -0,0 +1,9 @@
+.first-section {
+  text-align: start;
+}
+
+.first-text {
+  background-color: #f4f4f4;
+  padding: 48px 24px 40px 24px;
+  line-height: 25px;
```

Comment:
Here you had a margin-bottom 40px. I asked to remove it. I can see neither my comment nor the margin-bottom! If we remove it from here then you need the space to second section.

Reply:
I did not understand what you meant!